docs/guides/admin/docs/upgrade.md

```diff
@@ -1,143 +1,42 @@
+This guide describes how to upgrade Opencast 7.x to 8.x. In case you need information about how to upgrade older
 versions of Opencast, please refer to the [old release notes](https://docs.opencast.org).
 
 How to Upgrade
 --------------
 
 1. Stop your current Opencast instance
-2. Replace Opencast 6.x with 7.x
+2. Replace Opencast 7.x with 8.x
 3. Back-up Opencast files and database (optional)
 4. [Upgrade the database](#database-migration)
 5. [Upgrade the ActiveMQ configuration](#activemq-migration)
 6. Review the [configuration changes](#configuration-changes) and adjust your configuration accordingly
-7. Migrate the scheduled events
-8. [Optionally update the player links](#update-player-links)
-9. Rebuild the Elastic Search indexes
+7. Rebuild the Elastic Search indexes
 
 Database Migration
 ------------------
+Due to [MH-13514](https://opencast.jira.com/browse/MH-13514) a new column was added to one table.
 
-As part of performance optimizations, some tables were modified. This requires a database schema update. Also, one table
-is no longer needed and can be dropped. As with all database migrations, we strongly recommend to create a database
-backup before attempting the upgrade.
-
-You can find the database upgrade script in `docs/upgrade/6_to_7/`. This script is suitable for both, MariaDB and
+You can find the database upgrade script in `docs/upgrade/7_to_8/`. This script is suitable for both, MariaDB and
 MySQL.
 
 
 ActiveMQ Migration
 ------------------
-
-*So far, this is not required*
-
-
-Removal of Deprecated Access Control Ruleset
---------------------------------------------
-
-Opencast 7 finally removes the long-since deprecated `security/xacml` flavor for access control lists. This had not been
-used since before Opencast 1.2 (we could not track down its exact deprecation date due to its age). Additionally, all
-rule-sets which had been modified since had also been automnatically been updated to `security/xacml+series` which
-serves as replacement for the old flavor.
-
-In case Opencast still encounters such a rule set, it will now be ignored and access will be denied by default. A simple
-update of the permissions would fix this if that is required.
-
-Due to the extreme unlikeliness of anyone encountering this problem, there is no automatic migration. In case you run a
-system migrated from a pre-1.2 Matterhorn, you can make sure that there are no old rule-sets left using the following
-SQL queries:
-
-```sql
--- Check OAI-PMH publications:
-select * from oc_oaipmh_elements where flavor = 'security/xacml';
--- Check engage publications:
-select * from oc_search where mediapackage_xml like '%"security/xacml"%';
--- Check asset manager:
-select * from oc_assets_snapshot where mediapackage_xml like '%"security/xacml"%';
-```
+Due to [MH-13522](https://opencast.jira.com/browse/MH-13522) one new queue was added, so you have to update your
```

Comment:
@krinnewitz I think that is now obsolete and the PR does not need to be mentioned in the upgrade guide anymore.